django/db/models/indexes.py

```diff
@@ -123,10 +123,11 @@ def set_name_with_model(self, model):
             self.name = 'D%s' % self.name[1:]
 
     def __repr__(self):
-        return "<%s: fields='%s'%s%s>" % (
+        return "<%s: fields='%s'%s%s%s>" % (
             self.__class__.__name__, ', '.join(self.fields),
             '' if self.condition is None else ', condition=%s' % self.condition,
             '' if not self.include else ", include='%s'" % ', '.join(self.include),
+            '' if not self.opclasses else ' opclasses=%s' % ', '.join(self.opclasses),
```

Comment:
```suggestion
            '' if not self.opclasses else ", opclasses='%s'" % ', '.join(self.opclasses)
```

Reply:
I don't think we want the comma between the `fields` and `opclasses` _attributes_ do we? 🤔

Looking at the test case, I'd imagine this: 

```
<Index: fields='headline, body' opclasses='varchar_pattern_ops, text_pattern_ops'>
```

which **does** add the quotes, to be consistent with `fields`...

```suggestion
            '' if not self.opclasses else " opclasses='%s'" % ', '.join(self.opclasses),
```

but it's not HTML, so I'm not 100% sure what's desired.

Reply:
I added quotes and the comma.

Reply:
I added the comma to be consistent with the `include` and `condition` attributes. Also, on `ExclusionConstraint` we separate attributes by comma but not on `UniqueConstraint`, so 🤷 indeed..

Reply:
Right, I removed commas between attributes (we don't have them in most of custom `__repr__()`\`s).